drivers/hmis/lib/form_data/assessments/dummy_intake_assessment.json

```diff
@@ -41,58 +41,71 @@
-              "type": "BOOLEAN",
-              "link_id": "3.917.B",
-              "text": "Did the client stay less than 7 nights?",
-              "project_types_excluded": ["ES", "SERVICES_ONLY", "SH"],
-              "enable_behavior": "ALL",
-              "enable_when": [
-                {
-                  "question": "3.917.1",
-                  "operator": "EQUAL",
-                  "answer_group_code": "TEMPORARY_PERMANENT_OTHER"
-                }
+              "question": "3.917.2",
+              "operator": "IN",
+              "answer_codes": [
+                "LOS_90_DAYS_OR_MORE_BUT_LESS_THAN_ONE_YEAR",
+                "LOS_ONE_YEAR_OR_LONGER"
               ]
+            }
+          ]
+        },
+        {
+          "type": "DISPLAY",
+          "link_id": "3.917.B",
+          "text": "Client stayed 7+ days in a transitional or permanent housing situation. This is considered a \"break\" according to the HUD definition of chronic homelessness. Stopping data collection for 3.917 Prior Living Situation.",
+          "project_types_excluded": ["ES", "SERVICES_ONLY", "SH"],
+          "enable_behavior": "ALL",
+          "enable_when": [
+            {
+              "question": "3.917.1",
+              "operator": "EQUAL",
+              "answer_group_code": "TEMPORARY_PERMANENT_OTHER"
             },
             {
-              "type": "BOOLEAN",
-              "link_id": "3.917.C",
-              "text": "On the night before, did the client stay on the streets, ES or SH?",
-              "project_types_excluded": ["ES", "SERVICES_ONLY", "SH"],
-              "enable_behavior": "ANY",
-              "enable_when": [
-                {
-                  "question": "3.917.A",
-                  "operator": "EQUAL",
-                  "answer_boolean": true
-                },
-                {
-                  "question": "3.917.B",
-                  "operator": "EQUAL",
-                  "answer_boolean": true
-                }
+              "question": "3.917.2",
+              "operator": "IN",
+              "answer_codes": [
+                "LOS_ONE_WEEK_OR_MORE_BUT_LESS_THAN_ONE_MONTH",
+                "LOS_ONE_MONTH_OR_MORE_BUT_LESS_THAN_90_DAYS",
+                "LOS_90_DAYS_OR_MORE_BUT_LESS_THAN_ONE_YEAR",
+                "LOS_ONE_YEAR_OR_LONGER"
               ]
             }
           ]
         },
+        {
+          "type": "BOOLEAN",
+          "link_id": "3.917.C",
+          "text": "On the night before, did the client stay on the streets, ES or SH?",
+          "project_types_excluded": ["ES", "SERVICES_ONLY", "SH"],
+          "enable_behavior": "ALL",
+          "enable_when": [
+            {
+              "question": "3.917.A",
+              "operator": "ENABLED",
+              "answer_boolean": false
+            },
+            {
+              "question": "3.917.B",
+              "operator": "ENABLED",
+              "answer_boolean": false
+            }
+          ]
```

Comment:
This is the implementation of "Stop data collection for 3.917 Prior Living Situation."

This is saying: if either of the "stopping data collection..." display items are enabled, hide this.